daemon/src/db/cull_old_dlcs.rs

```diff
@@ -0,0 +1,308 @@
+use crate::db::Connection;
+use anyhow::Result;
+use model::EventKind;
+use sqlx::pool::PoolConnection;
+use sqlx::Sqlite;
+
+impl Connection {
+    #[allow(dead_code)]
```

Comment:
I assume this is just temporary and will be removed in a later commit. :)

Reply:
Good catch. You're right. I'm gonna undo this because it doesn't achieve much (there are other warnings after my recent refactor anyway).